.github/dependabot.yml

```
@@ -24,3 +24,15 @@ updates:
     reviewers:
       - "elastic/elastic-agent-control-plane"
     open-pull-requests-limit: 10
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - automation
+      - skip-changelog
+      - Team:Elastic-Agent
```

```suggestion
      - Team:Elastic-Agent-Control-Plane
```

If you are feeling ambitious you can make this change everywhere in this file.

---

Created https://github.com/elastic/elastic-agent/pull/4802